lib/matplotlib/axes/_axes.py

```diff
@@ -2694,106 +2694,101 @@ def errorbar(self, x, y, yerr=None, xerr=None,
             releases.
 
-          *barsabove*: [ *True* | *False* ]
-            if *True*, will plot the errorbars above the plot
+        barsabove: bool, optional, default: False
+            if True , will plot the errorbars above the plot
             symbols. Default is below.
 
-          *lolims* / *uplims* / *xlolims* / *xuplims*: [ *False* | *True* ]
+        lolims / uplims / xlolims / xuplims: bool, optional, default:None
             These arguments can be used to indicate that a value gives
             only upper/lower limits. In that case a caret symbol is
             used to indicate this. lims-arguments may be of the same
             type as *xerr* and *yerr*.  To use limits with inverted
             axes, :meth:`set_xlim` or :meth:`set_ylim` must be called
             before :meth:`errorbar`.
 
-          *errorevery*: positive integer
+        errorevery: positive integer, optional, default:1
             subsamples the errorbars. e.g., if errorevery=5, errorbars for
             every 5-th datapoint will be plotted. The data plot itself still
             shows all data points.
 
+        Returns
+        -------
+        plotline : :class:`~matplotlib.lines.Line2D` instance
+            x, y plot markers and/or line
+        caplines :
+            list of error bar cap :class:`~matplotlib.lines.Line2D` instances
+        barlinecols :
+            list of :class:`~matplotlib.collections.LineCollection` instances
+            for the horizontal and vertical error ranges.
+
+        Other Parameters
+        ----------------
         All other keyword arguments are passed on to the plot command for the
         markers. For example, this code makes big red squares with
         thick green edges::
 
-          x,y,yerr = rand(3,10)
-          errorbar(x, y, yerr, marker='s',
-                   mfc='red', mec='green', ms=20, mew=4)
+        x,y,yerr = rand(3,10)
+        errorbar(x, y, yerr, marker='s', mfc='red', mec='green', ms=20, mew=4)
 
-        where *mfc*, *mec*, *ms* and *mew* are aliases for the longer
-        property names, *markerfacecolor*, *markeredgecolor*, *markersize*
-        and *markeredgewidth*.
+        where mfc, mec, ms and mew are aliases for the longer
+        property names, markerfacecolor, markeredgecolor, markersize
+        and markeredgewidth.
 
         valid kwargs for the marker properties are
 
         %(Line2D)s
 
-        Returns (*plotline*, *caplines*, *barlinecols*):
-
-            *plotline*: :class:`~matplotlib.lines.Line2D` instance
-                *x*, *y* plot markers and/or line
-
-            *caplines*: list of error bar cap
-                :class:`~matplotlib.lines.Line2D` instances
-            *barlinecols*: list of
-                :class:`~matplotlib.collections.LineCollection` instances for
-                the horizontal and vertical error ranges.
-
-        **Example:**
 
+        Examples
+        -------
```

Comment:
You have a tiny problem here: there is a dash missing to underline the whole section.